### PR TITLE
[CORE-3693] Add 'status' property to companies list info

### PR DESCRIPTION
**Pull request by sanderbrugge**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1519,6 +1521,7 @@ Get a list of companies.
             + (object)
                 + id: `96a38bbf-24ed-4083-8a5c-20db92aa471e` (string)
                 + name: `Pied Piper` (string)
+                + status: `active` (string)
```
Comment (tiagobastos) [inline apiary.apib]: Let's be explicit and list all the variations that `status` can take:            
```suggestion
                + status: `active` (enum[string])
                    + Members
                        + `active`
                        + ...
```

File: apiary.apib [hunk 2]
```diff
@@ -399,6 +399,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### Latest
 
+- We added `status` on `companies.list`
+- We added `status` on `companies.info`
```
Comment (mrtus) [inline apiary.apib:403]: Q: Let's maybe create a new header for this and wrap the older changes under their month

Comment (mrtus) [reply apiary.apib:403]: Let's apply this change in the changelog file and not in the "build" file here though.

**Review by mrtus — COMMENTED** [review]
👍

File: apiary.apib (hunk 2 above)
Comment (JordyDeGroeve) [reply apiary.apib:403]: Oh missed that.

**Review by p-ba — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]
✅

**Review by JordyDeGroeve — APPROVED** [review]

**Pull request merged**